doc/code/qml_templates.rst

```diff
@@ -53,6 +53,17 @@ predefined pattern. You can import this function both via ``qml.broadcast`` and
 
     pennylane.broadcast
 
```

Comment:
In future one could think of merging the two sections to "helper functions/functions acting on templates" or something.